integrations/java/sync/workload-executor

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -o errexit
+
+workloadExecutorFile="/tmp/java-driver-workload-executor.json"
+
+rm -f "$workloadExecutorFile"
+
+# cat the workload executor JSON to a file
```

Comment:
It's easier to test the script in an IDE if the workload is passed indirectly as a file.  That's why we're not just passing the whole string as a command line parameter